lib/puppet/parser/functions/ensure_packages.rb

```diff
@@ -1,54 +0,0 @@
-# frozen_string_literal: true
-
-#
-# ensure_packages.rb
-#
-module Puppet::Parser::Functions
-  newfunction(:ensure_packages, type: :statement, doc: <<-DOC
```

Comment:
Interesting, I never knew that.

Reply:
Would that actually make it a possibly breaking change? Would people out there have used `ensure_packages ['mypackage']` and now need to write `ensure_packages(['mypackage'])`?

Reply:
It was a 'statement' function in so much that it didn't return anything.  It can't be called without parenthesis though as it doesn't appear in this list. https://github.com/puppetlabs/puppet/blob/ac45fe2b783708f64cb3a28f21ee92673e1f3216/lib/puppet/pops/model/factory.rb#L969

Reply:
Done.  I've also added a 3x shim.  I doubt it'll get any use for this function, so could drop it again if people prefer.  There are other functions (like ensure_resource) where it would possibly be more beneficial should we ever convert those.

Reply:
It was confirmed by @hlindberg that there is always a return value and being explicit in `nil` / `undef` is the correct way to go. In API v3 `type: :statement` did essentially the same.